tests/core/test_table_diff.py

```diff
@@ -134,8 +135,27 @@ def test_data_diff_decimals(sushi_context_fixed_date):
         on=["key"],
         decimals=4,
     )
-    assert diff.row_diff().full_match_count == 2
-    assert diff.row_diff().partial_match_count == 1
+
+    row_diff = diff.row_diff()
+    joined_sample_columns = row_diff.joined_sample.columns
+    assert row_diff.full_match_count == 2
+    assert row_diff.partial_match_count == 1
+    assert "s__value" in joined_sample_columns
+    assert "t__value" in joined_sample_columns
+
+    table_diff = TableDiff(
+        adapter=engine_adapter,
+        source="table_diff_source",
+        target="table_diff_target",
+        source_alias="dev",
```

Comment:
I thought this PR was for fixing the sample table when `source_alias` and `target_alias` aren't defined, but the test defines them here?

Reply:
Ah right, I missed that :) Thanks for clarifying!